airflow/www/static/js/utils/index.ts

```diff
@@ -185,6 +185,34 @@ const toSentenceCase = (camelCase: string): string => {
   return "";
 };
 
+const addColorKeyword = (
+  parsedLine: string,
+  errorKeywords: string[],
+  warningKeywords: string[]
+): string => {
+  const lowerParsedLine = parsedLine.toLowerCase();
+  const containsError = errorKeywords.some((keyword) =>
+    lowerParsedLine.includes(keyword)
+  );
+  const bold = (line: string) => `\x1b[1m${line}\x1b[0m`;
+  const red = (line: string) => `\x1b[31m${line}\x1b[39m`;
+  const yellow = (line: string) => `\x1b[33m${line}\x1b[39m`;
+
+  if (containsError) {
+    return bold(red(parsedLine));
+  }
+
+  const containsWarning = warningKeywords.some((keyword) =>
```

Comment:
Do you mind rearranging these variables a little bit? It was hard to read at first. Let's keep the containsWarning and containsError declarations near each other.

Reply:
I kept them away since if the error keyword is found I wanted to return the line red colored and don't want to search the line for warning keyword as an optimisation. Keeping them next to each other will make 2 searches for every line though only one is used in case of error keyword being present.